spec/aerospike/socket/ssl_spec.rb

```diff
@@ -17,6 +17,68 @@
 # License for the specific language governing permissions and limitations under
 # the License.
 
+RESOURCE_PATH = Pathname.new(__dir__).join('..', '..', 'resources')
+
+def resource(*path)
+  RESOURCE_PATH.join(*path).to_s
+end
+
 describe Aerospike::Socket::SSL do
-  pending
+  let(:ssl_options) { {} }
+
+  describe '::create_context' do
+    subject { described_class.create_context(ssl_options) }
+
+    it { is_expected.to be_a(OpenSSL::SSL::SSLContext) }
+  end
+
+  describe '::build_ssl_context' do
+    subject(:build_ssl_context) { described_class.build_ssl_context(ssl_options) }
+
+    context 'with pre-initialized context' do
+      let(:ssl_context) { double }
+      let(:ssl_options) { { context:  ssl_context } }
+
+      it { is_expected.to be ssl_context }
+    end
+
+    context 'without pre-initialized context' do
+      let(:ssl_options) { {} }
+
+      before do
+        allow(described_class).to receive(:create_context)
+        build_ssl_context
+      end
+
+      it { expect(described_class).to have_received(:create_context) }
+    end
+  end
+
+  describe '::create_context' do
+    subject(:create_context) { described_class.create_context(ssl_options) }
+
+    context 'when cert_file and pkey_file options are given' do
+      let(:cert) { resource('ssl', 'test.cert.pem') }
+      let(:pkey) { resource('ssl', 'test.key.pem') }
+      let(:ssl_options) { { cert_file: cert, pkey_file: pkey } }
+
+      before { allow_any_instance_of(OpenSSL::SSL::SSLContext).to receive(:add_certificate) }
```

Comment:
Change `before { ... }` to `before do ... end`? 'Cause I'm anal about this... ;-)